model/connection_string.go

```diff
@@ -190,7 +191,7 @@ func (cs ConnectionString) string(omit bool) string {
 		case crypto.String:
 			var value string
 			if omit {
-				value = base64.StdEncoding.EncodeToString([]byte(typ[:3])) + omitted
+				value = base64.StdEncoding.EncodeToString([]byte(typ[:3])) + omittedEllpsis
```

Comment:
Ellpsis - typo? I think it should be Ellipsis

Reply:
Yup, sorry. Force-pushed.